plan/stats.go

```diff
@@ -128,16 +128,27 @@ func (ds *DataSource) deriveStats() (*statsInfo, error) {
 	ds.statsAfterSelect = ds.getStatsByFilter(ds.pushedDownConds)
 	for _, path := range ds.possibleAccessPaths {
 		if path.isTablePath {
-			err := ds.deriveTablePathStats(path)
+			justPointRange, err := ds.deriveTablePathStats(path)
 			if err != nil {
 				return nil, errors.Trace(err)
 			}
+			// If there's only point range. Just remove other possible paths.
+			if justPointRange {
+				ds.possibleAccessPaths[0] = path
+				ds.possibleAccessPaths = ds.possibleAccessPaths[:1]
+				break
+			}
 			continue
 		}
-		err := ds.deriveIndexPathStats(path)
+		uniqueAndOnlyPoint, err := ds.deriveIndexPathStats(path)
```

Comment:
`pointQuery` is enough.

Reply:
I think we need to tell that this key is unique key.